setup.py

```diff
@@ -169,7 +169,7 @@ def run(self):
         # Ensure our temporary build directory exists
         try:
             os.makedirs(build_temp)
-        except IOError:
+        except:
```

Comment:
This doesn't look correct to me. What error is FreeBSD failing with, if not an `IOError`?